### PR TITLE
Test updates

### PR DESCRIPTION
**Pull request by derpy-duck**

### CODE REVIEW

File: example-2/evm/src/contracts/lendingHub/Hub.sol [hunk 1]
```diff
@@ -192,7 +196,10 @@ contract Hub is HubStructs, HubMessages, HubGetters, HubSetters, HubUtilities {
         checkValidAddress(assetAddress);
 
         // recheck if withdraw is valid given up to date prices? bc the prices can move in the time for VAA to come
-        require(allowedToWithdraw(withdrawer, assetAddress, amount), "Not enough in vault");
+        (bool check1, bool check2, bool check3) = allowedToWithdraw(withdrawer, assetAddress, amount);
+        require(check1, "Not enough in vault");
+        require(check2, "Not enough in global supply");
+        require(check3, "Vault is undercollateralized if this withdraw goes through");
```
Comment (wrinkledLatherer) [inline example-2/evm/src/contracts/lendingHub/Hub.sol:202]: this is great

File: example-2/evm/src/contracts/lendingHub/HubGetters.sol [hunk 2]
```diff
@@ -83,6 +83,10 @@ contract HubGetters is Context, HubStructs, HubState {
         return _state.interestAccrualIndexPrecision;
     }
 
+    function getMaxDecimals() internal view returns (uint8) {
+        return _state.MAX_DECIMALS;
```
Comment (wrinkledLatherer) [inline example-2/evm/src/contracts/lendingHub/HubGetters.sol:87]: i will update this to be an element of state that is updated whenever register happens

File: example-2/evm/src/contracts/lendingHub/HubUtilities.sol [hunk 3]
```diff
@@ -138,7 +138,7 @@ contract HubUtilities is Context, HubStructs, HubState, HubGetters, HubSetters {
         
         VaultAmount memory globalAmounts = denormalizeVaultAmount(getGlobalAmounts(assetAddress), assetAddress);
 
-        return (amounts.deposited - amounts.borrowed >= assetAmount) && (globalAmounts.deposited - globalAmounts.borrowed >= assetAmount) && (vaultDepositedValue - vaultBorrowedValue >= assetAmount * price; // / (10**assetInfo.decimals));
+        return ((amounts.deposited - amounts.borrowed >= assetAmount), (globalAmounts.deposited - globalAmounts.borrowed >= assetAmount), ((vaultDepositedValue - vaultBorrowedValue)*(10**assetInfo.decimals) >= assetAmount * price * (10 ** getMaxDecimals())));
```
Comment (wrinkledLatherer) [inline example-2/evm/src/contracts/lendingHub/HubUtilities.sol:141]: the last of these checks seems correct, but why not just multiply RHS by 10 ** (maxdecimals - assetinfo.decimals)? seems that would be clearer code to me?

Comment (derpy-duck) [reply example-2/evm/src/contracts/lendingHub/HubUtilities.sol:141]: you're right haha I wasn't thinking

**Pull request merged**